### PR TITLE
Regression Tests: Breadcrumb regression tests

### PR DESCRIPTION
**Pull request by gitjain**
#### Pull request checklist

- [ ] Addresses an existing issue: #0000
- [ ] Include a change request file if publishing 
- [ ] New feature, bugfix, or enhancement
  - [ ] Includes tests
- [ ] Documentation update

#### Description of changes

(give an overview)

#### Focus areas to test

(optional)

### CODE REVIEW

File: packages/office-ui-fabric-react/src/components/Breadcrumb/Breadcrumb.visualtest.ts [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+import { baseUrl } from '../../common/VisualTest';
+declare var phantomcss: IPhantomCSS;
+declare var casper: Casper;
+/* tslint:disable:no-function-expression */
+casper.start(baseUrl + 'breadcrumb').
+  then(function () {
+    phantomcss.screenshot('.Breadcrumb', 'Breadcrumb_not_pressed');
+  }).then(function () {
+    this.mouse.move('.Breadcrumb');
+    phantomcss.screenshot('.Breadcrumb', 'Breadcrumb_hovered');
```
Comment (micahgodbolt) [inline packages/office-ui-fabric-react/src/components/Breadcrumb/Breadcrumb.visualtest.ts]: I don't see hover doing anything. Do you need to hover over a specific part of the breadcrumb?

File: packages/office-ui-fabric-react/src/components/Breadcrumb/Breadcrumb.visualtest.ts [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+  }).then(function () {
+    this.mouse.down('.Breadcrumb');
```
Comment (micahgodbolt) [inline packages/office-ui-fabric-react/src/components/Breadcrumb/Breadcrumb.visualtest.ts]: Same here for pressed.

File: packages/office-ui-fabric-react/src/components/Breadcrumb/Breadcrumb.visualtest.ts (hunk 1 above)
Comment (gitjain) [inline packages/office-ui-fabric-react/src/components/Breadcrumb/Breadcrumb.visualtest.ts]: As png is captured with component id, hovering over a specific part and taking a png may not be possible.

Comment (micahgodbolt) [inline packages/office-ui-fabric-react/src/components/Breadcrumb/Breadcrumb.visualtest.ts]: @gitjain you can pass any selector you want into this.move.mouse. Even if you need to use nth-child or something, you should be able to target something to hover over and test.

**Comment by micahgodbolt** [comment]
@gitjain waiting to merge this one until you are able to update the baseline images for hover/click

**Pull request closed without merging**